assets/src/edit-story/components/colorPicker/colorPicker.js

```diff
@@ -96,6 +99,18 @@ function ColorPicker({
     },
   } = useColor();
 
+  const {
+    actions: { pushTransform },
+  } = useTransform();
+
+  const { selectedElementIds } = useStory(
```

Comment:
Should we perhaps pass this instead from the panels to keep the `colorPicker` more independent? There would be some prop-drilling but perhaps it makes more sense 🤷